lib/nerves_ssh.ex

```diff
@@ -105,6 +105,36 @@ defmodule NervesSSH do
     GenServer.call(name, {:remove_user, [user]})
   end
 
+  @doc """
+  Add a subsystem to the SSH daemon
+
+  If a subsystem with the same name already exists, it will be replaced.
+
+  **NOTE:** Requires OTP 25.1 or later.
```

Comment:
We've been dropping Elixir 1.13 support in other Nerves projects and the logger changes are making OTP 24 support hard anyway. How about we just assume OTP 25.1 or later so we don't have to maintain or worry about the runtime checks? It all looks good to me. Sorry I didn't get to share this earlier to save some time.

Reply:
Removed the runtime checks and added a commit dropping support for Elixir 1.13.